pkgs/by-name/wh/when/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenvNoCC
+, perl
+, installShellFiles
+, fetchFromBitbucket
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "when";
+  version = "1.1.46";
+
+  src = fetchFromBitbucket {
+    owner = "ben-crowell";
+    repo = "when";
+    rev = "a57ccbdf1d30e28a43a4bc5c68c37c171bb649ab";
```

Comment:
Ah, I hadn't checked whether there was a tagged release when reviewing, my bad.

Instead of a `let ... in` I would suggest the `finalAttrs idiom). I'm on mobile so it's hard to link to/write a suggestion. But please look it up in the nixpkgs manual.